### PR TITLE
Fix a regression in flashing the Dell dock

### PR DESCRIPTION
**Pull request by hughsie**
This fixes a regression caused by 15b668c, we should have been more careful.

Fixes https://github.com/fwupd/fwupd/issues/3732

Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [X] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: libfwupdplugin/fu-device.h [hunk 1]
```diff
@@ -378,6 +378,15 @@ typedef guint64 FuDeviceInternalFlags;
  */
 #define FU_DEVICE_INTERNAL_FLAG_USE_PARENT_FOR_BATTERY (1ull << 17)
 
+/**
+ * FU_DEVICE_INTERNAL_FLAG_USE_PROXY_FALLBACK:
+ *
+ * Use parent for the battery level and threshold.
+ *
+ * Since: 1.6.4
```
Comment (hughsie) [inline libfwupdplugin/fu-device.h:386]: this is because we want to backport it

**Review by superm1 — APPROVED** [review]

**Comment by hughsie** [comment]
Lets see if @CragW can confirm that it fixes the issue.

**Review by CragW — COMMENTED** [review]
Pls wait a second, doing more test cycles.

**Comment by superm1** [comment]
I think this is a correct commit, but maybe there is a secondary problem.

Crag, I'd suggest to run `fwupdtool` with `--verbose` on each run so you have the extra logging to point at where.

**Comment by CragW** [comment]
> I think this is a correct commit, but maybe there is a secondary problem.

Agree with you, this commit looks good to me. Let's go ahead and I will back here or new bug with further findings.

> Crag, I'd suggest to run `fwupdtool` with `--verbose` on each run so you have the extra logging to point at where.

Yeah, I should run verbose on each cycle...

**Comment by CragW** [comment]
I've just finished a couple more cycles and everything went fine, cheers.

**Pull request merged**